tests/lean/interactive/completionOption.lean

```diff
@@ -0,0 +1,2 @@
+set_option pp
```

Comment:
We should use a prefix that is less likely to break the test when new options are added. Perhaps `pp.proofs`?